recipes/pispy/recipe.yaml

```
@@ -0,0 +1,50 @@
+context:
+  name: pispy-client
+  version: "0.9.0"
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name | replace("-", "_") }}/${{ name | replace("-", "_") }}-${{ version }}.tar.gz
+  sha256: dff96b598412c6d051a173550a532c39180da42b389376ac38e0a517584a1faa
+
+build:
+  noarch: python
+  script: python -m pip install . -vv --no-deps --no-build-isolation
```

```suggestion
  script: python -m pip install . -vv
```

Let rattler-build just handle this:
https://github.com/prefix-dev/rattler-build/blob/7f0d61d199113442b5eab1e5c6c0f230dab606c9/src/env_vars.rs#L233-L242

---

Perfect! Forget that, right.